02-setup.md

```diff
@@ -48,9 +49,17 @@ He also has to set his favorite text editor, following this table:
 | Notepad++ (Win)    | `$ git config --global core.editor "'c:/program files (x86)/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
 | Kate (Linux)       | `$ git config --global core.editor "kate"`       |
 | Gedit (Linux)      | `$ git config --global core.editor "gedit -s -w"`   |
+| Scratch (Linux)       | `$ git config --global core.editor "scratch-text-editor"`  |
```

Comment:
Use of scratch is likely rare at this point, although it is becoming more popular as the default in elementary OS (a linux distro I recommend to beginners). I have mainly included it because the terminal command for Scratch is not immediately obvious to users of the GUI interface.

I realise this could snowball into a long list of everyone's favourites but this is one I've had trouble configuring myself and one I would recommend to new users (rather than `vim` or `emacs`).